### PR TITLE
Update to latest terracotta-apis

### PR DESCRIPTION
**Pull request by chrisdennis**

### CODE REVIEW

**Comment by chrisdennis** [comment]
What's the versioning story here? Am I right to be aligning the minor versions of APIs and Store or should I leave this at 10.7?

File: pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
 
   <groupId>org.terracotta.internal</groupId>
   <artifactId>tc-config-parent</artifactId>
-  <version>10.7-SNAPSHOT</version>
+  <version>10.9-SNAPSHOT</version>
```
Comment (cljohnso) [inline pom.xml:33]: 10.7 to 10.9?  Was the old one incorrect?

**Comment by cljohnso** [comment]
> What's the versioning story here? Am I right to be aligning the minor versions of APIs and Store or should I leave this at 10.7?

@myronkscott  needs to weigh in.

File: pom.xml (hunk 1 above)
Comment (chrisdennis) [reply pom.xml:33]: 🤷

Comment (myronkscott) [reply pom.xml:33]: why 10.9?  this thing just exists for conversion tooling.

**Comment by myronkscott** [comment]
> > What's the versioning story here? Am I right to be aligning the minor versions of APIs and Store or should I leave this at 10.7?
> 
> @myronkscott needs to weigh in.

This lives today just for backward compatibility.  The version is frozen because it is not changing.  If it were me, I'd just leave it at 10.7.

**Pull request merged**